docs/language/internals.rst

```diff
@@ -426,10 +429,10 @@ so our re-written ``nif`` would look like:
 
    (defmacro nif [expr pos-form zero-form neg-form]
```

Comment:
The version of `nif` you commented on here seems to work for me, both with this branch (`let-it-be`) and with master.